Source/CesiumRuntime/Private/Tests/CesiumLoadTest.cpp

```diff
@@ -299,15 +298,93 @@ bool RunLoadTest(const size_t locationIndex) {
-  return RunLoadTest(1);
+  return RunLoadTest(setupForDenver);
 }
 
 bool FCesiumLoadTestGoogleplex::RunTest(const FString& Parameters) {
-  return RunLoadTest(0);
+  return RunLoadTest(setupForGoogleTiles);
+}
+
+bool FCesiumLoadTestMontrealPointCloud::RunTest(const FString& Parameters) {
+  auto setup = [this](LoadTestContext& context) {
+    FVector targetOrigin(-73.616526, 45.57335, 95.048859);
+    FString ionToken(
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI2NmZhZTk4NS01MDFmLTRjODgtOTlkYy04NjIwODhiZWExOGYiLCJpZCI6MjU5LCJpYXQiOjE2ODg1MTI4ODd9.haoe5hsJyfHk1dQAHVK6N8dW_kfmtdbyuhlGwFdEHbM");
+
+    FCesiumCamera camera;
+    camera.ViewportSize = FVector2D(1024, 768);
+    camera.Location = FVector(0, 0, 0);
+    camera.Rotation = FRotator(-90.0, 0.0, 0.0);
+    camera.FieldOfViewDegrees = 90;
+    context.setCamera(camera);
+
+    context.georeference->SetGeoreferenceOriginLongitudeLatitudeHeight(
+        targetOrigin);
+
+    context.pawn->SetActorLocation(FVector(0, 0, 0));
+    context.pawn->SetActorRotation(FRotator(-90.0, 0.0, 0.0));
+
+    // Montreal Point Cloud
+    ACesium3DTileset* montrealTileset =
+        context.world->SpawnActor<ACesium3DTileset>();
+    montrealTileset->SetTilesetSource(ETilesetSource::FromCesiumIon);
+    montrealTileset->SetIonAssetID(28945);
+    montrealTileset->SetIonAccessToken(ionToken);
+    montrealTileset->SetMaximumScreenSpaceError(16.0);
+    montrealTileset->SetActorLabel(TEXT("Montreal Point Cloud"));
+
+    context.tilesets.push_back(montrealTileset);
+  };
+
+  auto after = [this](LoadTestContext& context) {
+    // Zoom way out
+    FCesiumCamera zoomedOut;
+    zoomedOut.ViewportSize = FVector2D(1024, 768);
+    zoomedOut.Location = FVector(0, 0, 7240000.0);
+    zoomedOut.Rotation = FRotator(-90.0, 0.0, 0.0);
+    zoomedOut.FieldOfViewDegrees = 90;
+    context.setCamera(zoomedOut);
+
+    context.pawn->SetActorLocation(zoomedOut.Location);
+
+    context.setSuspendUpdate(false);
+    tickWorldUntil(context, 10, breakWhenTilesetsLoaded);
+    context.setSuspendUpdate(true);
+
+    Cesium3DTilesSelection::Tileset* pTileset =
+        context.tilesets[0]->GetTileset();
+    if (TestNotNull("Tileset", pTileset)) {
+      int visibleTiles = 0;
+      pTileset->forEachLoadedTile([&](Cesium3DTilesSelection::Tile& tile) {
+        if (tile.getState() != Cesium3DTilesSelection::TileLoadState::Done)
+          return;
+        const Cesium3DTilesSelection::TileContent& content = tile.getContent();
+        const Cesium3DTilesSelection::TileRenderContent* pRenderContent =
+            content.getRenderContent();
+        if (!pRenderContent) {
+          return;
+        }
+
+        UCesiumGltfComponent* Gltf = static_cast<UCesiumGltfComponent*>(
+            pRenderContent->getRenderResources());
+        if (Gltf && Gltf->IsVisible()) {
+          ++visibleTiles;
+        }
+      });
+
+      TestEqual("visibleTiles", visibleTiles, 1);
```

Comment:
This unfortunately fails, and it calls attention to a bit of a problem in the performance testing framework. The problem is that the performance test is executed in an Editor world, which means that tiles are selected using any active editor viewports. That's a bit problem because different editor views will affect the performance, and in this particular case it also causes my test to fail because the editor viewport is zoomed in close and causes additional tiles to be selected. To see what I mean, switch your default Editor viewport to an orthographic view (click Perspective, then Top) and note that the test now passes.

In general, I worry that the way the performance test is executed here is just a little bit artificial, and that will affect our results in unknown ways. I think it would be better to use Play-in-Editor mode and let the Engine itself manage the ticking, by following the model for this that's used in the sublevel tests.

Reply:
Agreed. I never really liked how it just used an active Editor viewport. This needs to be more rigid and predictable.

Committed some changes to start / end PIE mode, which simultaneously
- Uses the additional camera (and viewport) that each location configures
- Fixes your test so that it passes
- Is less artificial. Calls typical in-game functions like ::BeginPlay ::BeginDestroy

Reply:
> I think it would be better to use Play-in-Editor mode and let the Engine itself manage the ticking, by following the model for this that's used in the **sublevel tests**.

Looks like the sublevel test uses two mechanisms for hooking into the game loop:

1) Executing tasks at the end of frame
![image](https://github.com/CesiumGS/cesium-unreal/assets/130494071/9b29ffa7-cb70-42a6-822e-3c8c2efd9834)

2) A special tick call in the main tick loop
![image](https://github.com/CesiumGS/cesium-unreal/assets/130494071/7e5deae6-90e9-48e4-bb6d-b491dd4e1340)

I haven't hooked into those yet for these reasons
1) I haven't got that far yet. The original test was made as quickly and simply as possible
2) I like the control of determining how often the tick loop executes, what tick times are used, and what is getting ticked
3) The inevitable next steps are to set up repeatable benchmarks and profile problems. If the way we tick doesn't impact either of these, do we need to change it?

But, it might be the case in the future that we MUST hook into the main tick loop for one reason for another. For example, what if we discover we need a test to render to a texture, but nothing is showing up because of this hacky tick logic?

Reply:
I'm still thinking on this btw. Stay tuned...

Reply:
Ok, committed some changes that do a more 'normal' tick loop, using latent commands. Kind of a blend of what Unreal recommends [here](https://docs.unrealengine.com/4.27/en-US/TestingAndOptimization/Automation/TechnicalGuide/), and the test helper functions you use for the sublevels test.